.husky/pre-commit

```diff
@@ -1,4 +1,5 @@
 #!/bin/sh
 . "$(dirname "$0")/_/husky.sh"
 
-yarn run docs && git add docs.md
+yarn run lint-staged
+yarn run docs && git add docs.md
```

Comment:
Just a question around husky. Is there any difference between creating this file vs adding something like this to `package.json` (besides that also running `docs && git add docs.md`)?
```
"husky": {
  "hooks": {
    "pre-commit": "lint-staged"
  }
},
```

Reply:
Here's the release notes: https://github.com/typicode/husky/releases/tag/v6.0.0

Reply:
Awesome, thanks!